### ISSUE TITLE
`Figure_1.png`

### ISSUE THREAD

**Issue by johanmalm** [johanmalm/footprint#1]
![Figure_1](https://github.com/user-attachments/assets/3a1f8ed6-17de-42a6-94af-b3e36a77fdce)

**Comment by tokyo4j** [comment]
Oh, I appear in two names: "Hiroaki Yamamoto" and "tokyo4j". I guess I've set different names in git configuration between laptops.

Anyway, nice visualization!

**Comment by johanmalm** [comment]
> Oh, I appear in two names: "Hiroaki Yamamoto" and "tokyo4j". I guess I've set different names in git configuration between laptops.
> 
> Anyway, nice visualization!

Yes, will try to merge them. @Consolatis showed me a method, but can't get it to work. Anyway, have a few other patches going in first :smile:

Will try to have a look at some your PR activity shortly.

**Comment by johanmalm** [comment]
![labwc](https://github.com/user-attachments/assets/c84d40a7-d8b3-4770-888e-21b71fd76d8b)

![wlroots](https://github.com/user-attachments/assets/d20e01e2-1f36-410f-8bef-be80c98bc275)
